src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -612,7 +628,7 @@ def martinize(input_pdb, output_path, skipss):
             n_l = line
         out.write(n_l)
     out.close()
-    subprocess.call(f"mv temp.pdb {cg_pdb_name}", shell=True)
+    if Path("temp.pdb").exists(): Path("temp.pdb").unlink()
```

Comment:
What about this version ?
```suggestion
    Path("temp.pdb").unlink(missing_ok=True)
```